### ISSUE TITLE
How to use pretrained embeddings provided by Stanford ?

### ISSUE THREAD

**Issue by KelleyYin** [pytorch/fairseq#194]
The format of glove pretrained embeddings is text file , which cant be used to initialize the model . Then, if used pretained embeddings, we also want to fix it .

**Comment by myleott** [comment]
Some models support pretrained embeddings, e.g., fconv: https://github.com/pytorch/fairseq/blob/1f08602f8f5adf7970e2b675fc50e72a637f8cad/fairseq/models/fconv.py#L38-L39.

Freezing should be as simple as adding `.detach` here: https://github.com/pytorch/fairseq/blob/1f08602f8f5adf7970e2b675fc50e72a637f8cad/fairseq/models/fconv.py#L435
